src/indev/lv_indev.c

```diff
@@ -486,11 +487,23 @@ lv_timer_t * lv_indev_get_read_timer(lv_indev_t * indev)
     return indev->read_timer;
 }
 
-void lv_indev_delete_read_timer(lv_indev_t * indev)
+void lv_indev_set_mode(lv_indev_t * indev, lv_indev_mode_t mode)
 {
-    if(indev && indev->read_timer) {
-        lv_timer_delete(indev->read_timer);
-        indev->read_timer = NULL;
+    if(!indev || indev->mode == mode)
+        return;
+
+    indev->mode = mode;
+    if(indev->read_timer) {
+        if(mode == LV_INDEV_MODE_EVENT) {
+            lv_timer_pause(indev->read_timer);
+            lv_timer_set_period(indev->read_timer, LV_INDEV_DEF_LONG_PRESS_REP_TIME);
```

Comment:
```suggestion
            lv_timer_set_period(indev->read_timer, LV_DEF_REFR_PERIOD);
```

Reply:
If we keep the default refresh rate, maybe we should remove `lv_timer_set_period`?